src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -119,6 +119,16 @@ the main body of the message as well as a quote.
 						@click.stop="handleReply">
 						{{ t('spreed', 'Reply') }}
 					</ActionButton>
+					<template
+						v-for="action in messageActions">
+						<ActionButton
+							:key="action.label"
+							:icon="action.icon"
+							:close-after-click="true"
+							@click="action.callback">
```

Comment:
```suggestion
							@click="action.callback(messageAPIData)">
```

So here we need to handover a public API message object (which we can not change frequently.

I would for the same of simplicity tend to just hand over the full object from the store so https://nextcloud-talk.readthedocs.io/en/latest/chat/#receive-chat-messages-of-a-conversation is the matching documentation

Reply:
oh sure

Reply:
In addition to the message data, details about the conversation (either in the message itself or as a separate object) would probably also be good to have at hand as the integrating app.

Reply:
What sort of info? Would the conversation token be enough?

Reply:
token and title would be ideal i'd say for being able to immediately show some context as the action and generating a link to it.